packages/renderer/src/lib/table/table.ts

```diff
@@ -47,6 +47,19 @@ export interface ColumnInformation<Type> {
    * @param comparator
    */
   readonly comparator?: (object1: Type, object2: Type) => number;
+
+  /**
+   * The 'natural' or initial sort direction. Most columns are
+   * naturally sorted in ascending order and do not need to
+   * specify this value - e.g. names are sorted alphabetically.
+   *
+   * Columns that are naturally sorted in descending order -
+   * e.g. file sizes or 'number of children' by biggest first -
+   * can set this value to change the initial sort direction.
+   *
+   * Defaults to 'ascending'.
+   */
+  readonly initialSortDir?: 'ascending' | 'descending';
```

Comment:
could be just be named `order` ?  (I mean, of course user can change it later but when we register objects we specify an order) (or then `initialOrder`

Reply:
Renamed to initialOrder in new commit.